pkg/operator/controller/ingress/load_balancer_service_test.go

```diff
@@ -779,6 +820,7 @@ func Test_desiredLoadBalancerService(t *testing.T) {
 				TargetPort: intstr.FromString("https"),
 			}}, svc.Spec.Ports)
 			assert.Equal(t, "None", string(svc.Spec.SessionAffinity))
+			assert.Equal(t, tc.expectedFloatingIP, svc.Spec.LoadBalancerIP)
 		})
```

Comment:
Mind adding a test case to `Test_loadBalancerServiceChanged` to cover the fact that updates to `spec.LoadBalancerIP` should NOT cause a change? (now that we are using `spec.LoadBalancerIP`.

```
		{
			description: "if .spec.loadBalancerIP changes",
			mutate: func(svc *corev1.Service) {
				svc.Spec.LoadBalancerIP = "3.4.5.6"
			},
			expect: false,
		},
```